src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -16,46 +18,57 @@ import FastImage from 'react-native-fast-image';
  *
  * @param {ProfilePicturePropsParam} props
  */
-const ProfilePicture = ({ onImageContainerClick, profilePicPath, disabledAddIcon = false}) => {
-    const __renderAddIcon = () => {
-        if(!profilePicPath || !disabledAddIcon) return <></>
 
-        return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />
-    }
-    return <View style={styles.wrapImageProfile}>
-        <TouchableNativeFeedback onPress={onImageContainerClick}>
-            <View style={styles.profileImageContainer}>
+const ProfilePicture = ({
+  onImageContainerClick,
+  profilePicPath,
+  disabledAddIcon = false,
+  karmaScore = 0
+}) => {
+  const renderAddIcon = () => {
+    if (!profilePicPath || !disabledAddIcon) return <></>;
+
+    return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />;
+  };
+  return (
+    <View style={styles.wrapImageProfile}>
+      <TouchableNativeFeedback onPress={onImageContainerClick}>
+        <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(2)}>
+          <View style={styles.profileImageContainer}>
             <Image
-                style={styles.profileImage}
-                source={{
-                uri: profilePicPath
-                    ? `${profilePicPath}`
-                    : DEFAULT_PROFILE_PIC_PATH,
-                }}
-                resizeMode={FastImage.resizeMode.stretch}
+              style={styles.profileImage}
+              source={{
+                uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH
+              }}
+              resizeMode={FastImage.resizeMode.stretch}
             />
-            { __renderAddIcon() }
-            </View>
-        </TouchableNativeFeedback>
+            {renderAddIcon()}
+          </View>
+        </CircleGradient>
+      </TouchableNativeFeedback>
     </View>
-}
+  );
+};
 
 let styles = StyleSheet.create({
-    addCircle: {position: 'absolute', top: 25, left: 25},
-    profileImage: {
-        width: 100,
-        height: 100,
-        borderRadius: 100,
-        // marginBottom: 12
-    },
-    profileImageContainer: {
-        width: 100,
-        borderRadius: 100,
-    },
-    wrapImageProfile: {
-        marginTop: 14,
-        flexDirection: 'column',
-    },
-})
+  addCircle: {position: 'absolute', top: 25, left: 25},
+  profileImage: {
+    width: 97,
+    height: 97,
+    borderRadius: 100,
+    marginLeft: 1.4,
+    marginTop: 2,
+    marginBottom: 2
+  },
+  profileImageContainer: {
+    width: 100,
+    borderRadius: 100
+    // zIndex: 1000
+  },
+  wrapImageProfile: {
+    // marginTop: 14,
+    flexDirection: 'column'
+  }
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles for the `profileImage` have been modified. The width and height are reduced by 3 units and the `marginLeft`, `marginTop`, and `marginBottom` properties are added. These changes seem to adjust the positioning of the profile image within the circular gradient background. However, it's important to ensure these changes don't negatively impact the layout on different screen sizes or resolutions.

```diff
-    profileImage: {
-        width: 100,
-        height: 100,
-        borderRadius: 100,
-        // marginBottom: 12
-    },
+    profileImage: {
+        width: 97,
+        height: 97,
+        borderRadius: 100,
+        marginLeft: 1.4,
+        marginTop: 2,
+        marginBottom: 2
+    },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->